webui/react/src/e2e/models/ant/Dropdown.ts

```diff
@@ -47,7 +47,7 @@ export class Dropdown extends BaseComponent {
           // either provide a childNode or replace this method.
           throw new Error('This dropdown does not have a child node to click on.');
         }
-        await this.childNode.pwLocator.click();
+        await this.childNode.pwLocator.click({ timeout: 15_000 }); // refreshing with 10s intervals, this should give us enough time for everything to be stable even if a refresh occurs in the first 5 seconds.
```

Comment:
we should give args to `openMethod` and pass the args through to this click

Reply:
added, but I left the default as 15. I think that makes sense since the 10s refresh seems to happen basically everywhere.